sdk/template/template/src/configurationClient.ts

```diff
@@ -0,0 +1,134 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import {
+  PipelineOptions,
+  TokenCredential,
+  OperationOptions,
+  bearerTokenAuthenticationPolicy,
+  createPipelineFromOptions,
+  InternalPipelineOptions
+} from "@azure/core-http";
```

Comment:
should we move it to `core-https`? cc @xirzec

Reply:
I think that would be good

Reply:
Let's take that as an issue instead of working it into this PR.

Reply:
can we open an issue to track this?